aiopyrq/unique_queues.py

```diff
@@ -359,6 +384,28 @@ def get():
             return items
             """
 
+        @staticmethod
+        def delete():
+            """
+            :return: LUA Script for DELETE command
+            """
+            return """
+            local queue = KEYS[1]
+            local set = KEYS[2]
+            local processing = KEYS[3]
+            local timeouts = KEYS[4]
+            local time = ARGV[1]
+            local item = ARGV[2]
+            redis.call('hset', timeouts, processing, time)
+            while true do
```

Comment:
nejsem si jistu, zda v pripade unique queue je potreba delat cyklus. Vzdy by tam mel byt item jen jednou, coz je zajisteno i tim setem ... ale asi to nicemu nevadi a kdyby se nejak vyskytl sotek, ze by se tam dostal item nekolikrat, tak asi je to pro jistotu lepsi ...